pages/osx/bless.md

```diff
@@ -0,0 +1,19 @@
+# bless
+
+> Set volume bootability and startup disk options.
+
+- FOLDER MODE, To bless a volume with only Mac OS X or Darwin, and create the BootX and boot.efi files as needed:
```

Comment:
Perhaps this will read better?
```suggestion
- FOLDER MODE: Bless a volume with only Mac OS X or Darwin, and create the BootX and boot.efi files as needed:
```

What does `blessing` a volume actually do?